app/src/main/res/layout/activity_main3.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
If any layout contains only one child (`RecyclerView` in this case), we can omit it. By reducing number of nested views / view groups, we improve performance.
So we can remove `FrameLayout` here and leave only recycler view.